### PR TITLE
Contributor workshop update mentoring information

### PR DESCRIPTION
**Pull request by alisondy**
In line with moving the contributor workshop to a self paced course,
we need to provide resources on how to contribute and update the workshop.

This PR does the following
* Update the Video Guide
* Create Guide on how to add segments to the workshop
* Create a Glossary to define repeated verbalage (segments, exercises)
* Outline how the workshop is structured
* Create template for segments
* Move existing resources for running live NCW to ``live-workshop``

Related PR
https://github.com/kubernetes/contributor-site/pull/220

### CODE REVIEW

File: mentoring/programs/contributor-workshop/resources/guides/videoguide.md [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+
+You can use any format/style as long as the same information in the guide is captured in the video. Slides, A screenshare, greenscreen yourself into the repo even! Be creative.
+
+## Encoding Guidelines
+
+### Zoom Recording
+Handled by zoom, Can't change these settings.
+
+### OBS
+(todo : match with media encoder)
```
Comment (alisondy) [inline mentoring/programs/contributor-workshop/resources/guides/videoguide.md:67]: make this a bulletted list

Comment (alisondy) [reply mentoring/programs/contributor-workshop/resources/guides/videoguide.md:67]: image

File: mentoring/programs/contributor-workshop/resources/guides/videoguide.md [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+[GitHub Link](https://github.com/dralletje/Windowed)
+
+#### Start the meeting
+
+Start a meeting in zoom (you can start and record a meeting with a single participant)
+
+#### Setting up slides for recording
+
+Open your slides in Presenter View
+![Presenter View](presenterview.png)
```
Comment (alisondy) [inline mentoring/programs/contributor-workshop/resources/guides/videoguide.md:100]: new line needed here

File: mentoring/programs/contributor-workshop/resources/guides/videoguide.md [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+##### Update the thumbnail image
+
+Update the Thumbnail image on the previous video to
+
+insert-deprecation-thumbnail-here.jpeg
+```txt
+
+
+THIS VIDEO IS DEPRECATED!
+See the comments & description for details on where to find the new video guide!
```
Comment (alisondy) [inline mentoring/programs/contributor-workshop/resources/guides/videoguide.md:273]: There may not be a new video, see the comments and description for more information

File: mentoring/programs/contributor-workshop/resources/guides/videoguide.md [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+
+
+```
+
+##### Update the Description
+
+On the previous video, add the following to the start of the description
+```txt
```
Comment (alisondy) [inline mentoring/programs/contributor-workshop/resources/guides/videoguide.md:281]: new line

File: mentoring/programs/contributor-workshop/resources/templates/segment-template.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Segment Name"   # The name of the page
+weight: 0               # Ordering of the page
```
Comment (alisondy) [inline mentoring/programs/contributor-workshop/resources/templates/segment-template.md:3]: can't have a weight of 0 that is invalid

**Review by mrbobbytables — COMMENTED** [review]
Thanks! I made a few suggestions
Also - not to be super nitty, but is there anything else planned for the root of the contributor-workshop dir? If not, we can probably move everything in resources up a level to the contributor-workshop dir directly.

I think that was largely an artifact of when it was role under events^^;;;

**Comment by alisondy** [comment]
i messed up git 😨

File: mentoring/programs/contributor-workshop/resources/guides/updating-segment.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+```bash
+git checkout workshop
+
+git checkout -b my-changes
+```
+
+### Run the site locally with your changes
+
+Follow the instructions in the top level [``README.md``](https://github.com/kubernetes/contributor-site/tree/master/README.md#running-the-site-locally) on the contributor-site.
```
Comment (SergeyKanzhelev) [inline mentoring/programs/contributor-workshop/resources/guides/updating-segment.md:33]: curious if it supposed to be in master?
```suggestion
Follow the instructions in the top level [``README.md``](https://github.com/kubernetes/contributor-site/tree/workshop/README.md#running-the-site-locally) on the contributor-site.
```

**Review by SergeyKanzhelev — COMMENTED** [review]
minor neats

File: mentoring/programs/contributor-workshop/resources/guides/videoguide.md [hunk 7]
```diff
@@ -0,0 +1,397 @@
...
+
+```txt
+## Video Walkthrough
+
+{{< youtube id="<id of your video>" title="Contributor Workshop : <Segment Name" >}}
+```
+
+## Best Practices & Recommendations
+
+### Video
```
[the remark this answers is not in the document]
Comment (masagej0548672680) [reply mentoring/programs/contributor-workshop/resources/guides/videoguide.md:212]: هلا ان شعرك جميل

**Comment by alisondy** [comment]
/remove-sig apps
/remove-sig architecture
/remove-sig auth
/remove-sig autoscaling
/remove-sig cli
/remove-sig cloud-provider
/remove-sig cluster-lifecycle
/remove-sig docs
/remove-sig instrumentation
/remove-sig multicluster
/remove-sig network
/remove-sig node
/remove-sig release
/remove-sig scalability
/remove-sig scheduling
/remove-sig security
/remove-sig storage
/remove-sig testing
/remove-sig ui
/remove-sig usability
/remove-wg api-expression
/remove-wg k8s-infra
/remove-wg multitenancy
/remove-wg policy

tidy up labels

**Comment by jberkus** [comment]
/lgtm
/approve

let's get this merged, it's going to be easier to work on if people can share it.

**Pull request merged**